tests/standard_to_enhanced/icone_translator_test.py

```diff
@@ -165,45 +165,46 @@ def test_wzdx_creator(mockuuid):
 
     expected_wzdx = {
```

Comment:
Should we pull this file's JSON declarations into separate files like in our other tests?

Reply:
moved to icone_translator_data.py